samples/boards/nrf52/mesh/onoff_level_app/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(QEMU_EXTRA_FLAGS -s)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_link_libraries(app subsys__bluetooth)
+
+target_sources(app PRIVATE
+	       src/main.c
+	       src/common.c
+	       src/mesh/ble_mesh.c
+               src/mesh/device_composition.c
+               src/mesh/publisher.c
```

Comment:
Minor, but looks like the above two lines have different indentation than the others.